### PR TITLE
How to check and change hostname of debian 9

### PR DESCRIPTION
**Pull request by yfstein**
Here i wrote a little beginners guide to check and change the hostname of a debian 9 system.

I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: yFStein info@meikelbloch.de

### CODE REVIEW

File: tutorials/check-hostname-and-change-it-on-debian-9/01.en.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/check-hostname-and-change-it-on-debian-9"
+slug: "check-hostname-and-change-it-on-debian-9"
+date: "2019-05-03"
+title: "Check hostname and change it on debian 9"
+short_description: "This Tutorial explain how to check the hostname and change it on debian 9"
```
Comment (n-se) [inline tutorials/check-hostname-and-change-it-on-debian-9/01.en.md]:
```suggestion
short_description: "This tutorial explain how to check the hostname and change it on debian 9"
```

File: tutorials/check-hostname-and-change-it-on-debian-9/01.en.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+**Prerequisites**
+
+To check and change the hostname on your server, make sure you are logged into your server with a sudo user.
+
+This guide is based on a fresh installed debian 9 cloud server.
+
+## Step 1 - Check the actual hostname
+
+To check the actual hostname, u have three different ways. For the first u can type:
```
Comment (n-se) [inline tutorials/check-hostname-and-change-it-on-debian-9/01.en.md]: Do not use `u` here, please. Better use a "normal" language, `you`.

File: tutorials/check-hostname-and-change-it-on-debian-9/01.en.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+```sudo cat /etc/hosts```
```
Comment (n-se) [inline tutorials/check-hostname-and-change-it-on-debian-9/01.en.md]: For a "one-liner", please only use a single ` at the beginning and at the end of a line.

File: tutorials/check-hostname-and-change-it-on-debian-9/01.en.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+where the last option show you the configured fully qualified hostname.
+
+## Step 2 - Change the actual hostname
+
+Also when changing the hostname we have different possibilities. The simplest method would be this option:
+
+```sudo hostnamectl set-hostname servername.fqdn.tld```
+
+Here u just need to change servername.fqdn.tld to your servers fully qualified hostname, and the daemon will done the rest. Then we restart the SSH service and the new hostname is set:
```
Comment (n-se) [inline tutorials/check-hostname-and-change-it-on-debian-9/01.en.md]: > fully qualified hostname

What is a fully qualified hostname? I would like to see some more explanation about this one here.

Comment (yfstein) [reply tutorials/check-hostname-and-change-it-on-debian-9/01.en.md]: Hey thanks for the fast checkup. Sure - i changed a bit the typo and formatting and got some extra explanations. Did u miss anything else?

**Comment by yfstein** [comment]
@n-se i done some typos and formatting. Also got some more explanations about the hostname. Any further i should add or change?

**Pull request merged**